src/app/courses/courses.component.html

```diff
@@ -1,27 +1,72 @@
 <mat-toolbar *ngIf="!isForm">
-  <button mat-icon-button (click)="goBack()" *ngIf="!isDialog"><mat-icon>arrow_back</mat-icon></button>
-  <span i18n>{myCoursesFilter.value, select, on {myCourses} off {Courses}}</span>
-  <mat-form-field class="font-size-1 margin-lr-4 mat-form-field-type-no-underline mat-form-field-dynamic-width collections-search">
-    <planet-tag-input [formControl]="tagFilter" [db]="dbName" [parent]="parent" [filteredData]="courses.filteredData" [helperText]="false" [largeFont]="true" [selectMany]="false" mode="filter" [updateRouteParam]="!isDialog"></planet-tag-input>
-  </mat-form-field>
-  <span class="toolbar-fill"></span>
+  <ng-container *ngIf="deviceType !== deviceTypes.MOBILE && deviceType !== deviceTypes.TABLET; else mobileOrTabletView">
+    <mat-toolbar-row>
+      <button mat-icon-button (click)="goBack()" *ngIf="!isDialog"><mat-icon>arrow_back</mat-icon></button>
```

Comment:
As this whole block is repeating may be make it subtemplate of it and call it instead of repeating twice.